examples/serve/multi_node/multi_node.yaml

```diff
@@ -0,0 +1,49 @@
+# SkyServe YAML to launch a service where each replica contains multiple nodes.
+# Preemption of one node will cause the replica to be preempted and new replica to be launched.
+
+service:
+  readiness_probe: 
+    path: /v1/models
+    initial_delay_seconds: 6000
+  replicas: 1
+
+envs:
+  MODEL_NAME: meta-llama/Llama-2-70b-hf
+  HF_TOKEN: <your-huggingface-token> # TODO: Replace with huggingface token
+
+num_nodes: 2
+
+resources:
+  cloud: gcp
+  ports: 8000
+  accelerators: A100:2
+  use_spot: true
+  memory: 64+
+
+setup: |
+  conda create -n vllm python=3.11 -y
+  conda activate vllm
+  pip install -U "ray[default]"
```

Comment:
it will be great to pin down an version to keep everything works properly. in most cases simply ssh to the launched cluster and use `pip show ray` and use that version is enough